index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    />
+
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <!-- 헤더 -->
+    <header class="header inner">
+      <a class="logo" href="/">
+        <div class="logoImg">
+          <img src="img/판다 얼굴.svg" />
```

Comment:
image 의 alt 속성이 빠져있네요! alt 속성은 이미지가 렌더링되지 못할 때 나타날 문자열을 지정하기 위한 값으로 웹 접근성과 SEO 에 영향을 줄 수 있습니다. 웹 표준기관 WHATWG 의 대체 텍스트 제공 가이드라인 에도 다음과 같은 문구가 있습니다.

"Except where otherwise specified, the alt attribute must be specified and its value must not be empty"
달리 지정된 경우를 제외하고 alt 특성을 지정해야만 하며 해당 값은 비워 둘 수 없습니다.

alt 속성과 관련된 문서도 첨부드리니 참고해보세요 : )
https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt